cmd/collector/app/http_handler.go

```diff
@@ -109,26 +92,3 @@ func (aH *APIHandler) saveSpan(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
```

Comment:
I actually prefer returning 202 like you've done in the zipkin http handler, mind making the change here too?